cmd/infra/azure/create.go

```diff
@@ -244,7 +261,7 @@ func (o *CreateInfraOptions) Run(ctx context.Context, l logr.Logger) (*CreateInf
 }
 
 // createResourceGroup creates the Azure resource group used to group all Azure infrastructure resources
-func createResourceGroup(ctx context.Context, o *CreateInfraOptions, azureCreds azcore.TokenCredential, subscriptionID string) (string, string, string, error) {
+func createResourceGroup(ctx context.Context, o *CreateInfraOptions, azureCreds azcore.TokenCredential, rgName, subscriptionID string) (string, string, string, error) {
```

Comment:
Is the comment of this signature true any more? can we at minimum update that to reflect reality and how this parameter impacts the output?

Reply:
Done